Processing/BRCA_Deep_Dive.Rmd

```diff
@@ -2175,22 +1841,849 @@ ggplot() +
+                        pull(spei_hot_dry), 0.2)
+
+pump_years_below_12ft_hd <- spei_hot_dry2 %>% dplyr::filter(year > 2000) %>%
+               dplyr::mutate(
+                 pump_depth = 
+                    ifelse(spei_hot_dry < threshold,
+                           static_ft_predict-5.5,static_ft_predict)) %>%
+  dplyr::select(year,pump_depth) %>%
+  group_by(year) %>%
+  dplyr::summarize(pump_depth = min(pump_depth)) %>%
+  dplyr::filter(pump_depth <= -12)
+
+pump_years_below_12ft_ww <- spei_warm_wet2 %>% dplyr::filter(year > 2000) %>%
+               dplyr::mutate(
+                 pump_depth = 
+                    ifelse(spei_warm_wet < threshold,
+                           static_ft_predict-5.5,static_ft_predict)) %>%
+  dplyr::select(year,pump_depth) %>%
+  group_by(year) %>%
+  dplyr::summarize(pump_depth = min(pump_depth)) %>%
+  dplyr::filter(pump_depth <= -12)
+
+```
+
+##### - Table 3
+
+```{r table_3}
+Lev_summary <- bind_rows(
+  spei_historical %>%
+    dplyr::filter(year(time_ag) < 2011) %>%
+  dplyr::select(time_ag, static_ft) %>%
+  dplyr::summarize(period = "Historical",
+                     mean = mean(static_ft, na.rm = TRUE),
+                     median = median(static_ft, na.rm = TRUE),
+                     cv = sd(static_ft, na.rm = TRUE) / mean,
+                     max = max(static_ft, na.rm = TRUE),
+                     min = min(static_ft,na.rm = TRUE),
+                     q90 = quantile(static_ft, 0.9, na.rm = TRUE),
+                     q10 = quantile(static_ft, 0.1, na.rm = TRUE)),
+  spei_warm_wet2 %>%
+    dplyr::filter(year(time_ag) > 2039) %>%
+  dplyr::select(time_ag, static_ft_predict) %>%
+  dplyr::summarize(period = "Warm Wet",
+                     mean = mean(static_ft_predict, na.rm = TRUE),
+                     median = median(static_ft_predict, na.rm = TRUE),
+                     cv = sd(static_ft_predict, na.rm = TRUE) / mean,
+                     max = max(static_ft_predict, na.rm = TRUE),
+                     min = min(static_ft_predict,na.rm = TRUE),
+                     q90 = quantile(static_ft_predict, 0.9, na.rm = TRUE),
+                     q10 = quantile(static_ft_predict, 0.1, na.rm = TRUE)),
+  spei_hot_dry2 %>%
+     dplyr::filter(year(time_ag) > 2039) %>%
+  dplyr::select(time_ag, static_ft_predict) %>%
+  dplyr::summarize(period = "Hot Dry",
+                     mean = mean(static_ft_predict, na.rm = TRUE),
+                     median = median(static_ft_predict, na.rm = TRUE),
+                     cv = sd(static_ft_predict, na.rm = TRUE) / mean,
+                     max = max(static_ft_predict, na.rm = TRUE),
+                     min = min(static_ft_predict,na.rm = TRUE),
+                     q90 = quantile(static_ft_predict, 0.9, na.rm = TRUE),
+                     q10 = quantile(static_ft_predict, 0.1, na.rm = TRUE)))
+
+
+kable(Lev_summary)
+
+```
+
+Project gw volume into future
+
+```{r}
+
+avg_thx_low <- 60
+avg_thx_mid <- 60
```

Comment:
Yes - I kept them as separate assignments because I previously tested multiple average thicknesses. But, it creates such a wide spread in volume results that it ends up just making the whole effort pointless.  However, in case we go back to testing different depths, which I could foresee happening based on Tyler's comments about everyone being familiar with the work of Marine, I'm going to keep it for now.